layouts/dataPages.vue

```diff
@@ -0,0 +1,316 @@
+<template>
+    <header :class="{ 'scrolled-nav': scrollNav }">
+        <div class="logo">
+            <span>uLogo</span>
+        </div>
+        
+        <nav>
+            <ul v-show="!Tablet" class="navigation">
+                <li>
+                    <NuxtLink  class="link" to="/">Home</NuxtLink >
+                </li>
+                <li>
+                    <NuxtLink class="link" to="/edgeBoosts">edge boosts</NuxtLink >
+                </li>
+                <li>
+                    <NuxtLink  class="link" to="/edgeActions">edge actions</NuxtLink>
+                </li>
+                <li>
+                    <NuxtLink  class="link" to="/rules">rules</NuxtLink >
+                </li>
+                <li>
+                    <NuxtLink  class="link" to="/homeBrew">homebrew</NuxtLink >
+                </li>
+            </ul>
+            <div class="berger-Menu">
```

Comment:
berger sollte in burger umbenannt werden
da bei wartung das leichter zu fehlern führen kann

Reply:
Fixed